Add support for `--no-wrapper` on `CreateProjectMojo`

### Description

When creating an application with the quarkus cli, it is possible to not generate the maven wrapper.
this is particularly useful when using the quarkus agent mcp, as it fails with the following message: `"Wrapper script '" + wrapper.getAbsolutePath() + "' is NOT tracked by git. + "It could be malicious. Add it to git or use the system-installed build tool.")` 
however, this option does not seem to be available on the quarkus maven plugin https://github.com/quarkusio/quarkus/blob/main/devtools/maven/src/main/java/io/quarkus/maven/CreateProjectMojo.java
This issue is about adding a `@Parameter(property = "noWrapper")` on `CreateProjectMojo` to be at the same level as the CLI.
cc @aloubyansky 

### Implementation ideas

_No response_

the other reason we do not want the mvn wrapper, is that we have our own maven installation. we do not want to use the wrapper. and if the files are there, it will make the cli fail on some commands.
so if I create an app with `quarkus create app toto -x rest -x <our extensions>`
then I can compile it with `mvn package` (our `mvn`, not the wrapper).
after that `mvn quarkus:dev` is OK, but `quarkus dev` will fail with:
```
WARNING: A terminally deprecated method in sun.misc.Unsafe has been called
WARNING: sun.misc.Unsafe::staticFieldBase has been called by com.google.inject.internal.aop.HiddenClassDefiner (file:/C:/Users/Sevel/.m2/wrapper/dists/apache-maven-3.9.11/03d7e36a140982eea48e22c1dcac01d8862b2550b2939e09a0809bbc5182a5bc/lib/guice-5.1.0-classes.jar)
WARNING: Please consider reporting this to the maintainers of class com.google.inject.internal.aop.HiddenClassDefiner
WARNING: sun.misc.Unsafe::staticFieldBase will be removed in a future release
[INFO] Scanning for projects...
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[FATAL] Non-resolvable parent POM for org.acme:toto:1.0.0-SNAPSHOT: The following artifacts could not be resolved: com.x.y.quarkus:x-quarkus-parent:pom:XX (absent): com.x.y.quarkus:x-quarkus-parent:pom:XX was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced and 'parent.relativePath' points at wrong local POM @ line 5, column 13
 @
[ERROR] The build could not read 1 project -> [Help 1]
[ERROR]
[ERROR]   The project org.acme:toto:1.0.0-SNAPSHOT (E:\tmp\test\toto\pom.xml) has 1 error
[ERROR]     Non-resolvable parent POM for org.acme:toto:1.0.0-SNAPSHOT: The following artifacts could not be resolved: com.x.y.quarkus:x-quarkus-parent:pom:XX (absent): com.x.y.quarkus:x-quarkus-parent:pom:XX was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced and 'parent.relativePath' points at wrong local POM @ line 5, column 13 -> [Help 2]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectBuildingException
[ERROR] [Help 2] http://cwiki.apache.org/confluence/display/MAVEN/UnresolvableModelException
2026-06-08 16:45:52,521 WARN  [io.smallrye.common.process] (process-waiter-"mvnw.cmd"-27108) SRCOM05000: Command E:\tmp\test\toto\mvnw.cmd (pid 27108) completed successfully but logged on stderr:
        WARNING: A terminally deprecated method in sun.misc.Unsafe has been called
        WARNING: sun.misc.Unsafe::staticFieldBase has been called by com.google.inject.internal.aop.HiddenClassDefiner (file:/C:/Users/Sevel/.m2/wrapper/dists/apache-maven-3.9.11/03d7e36a140982eea48e22c1dcac01d8862b2550b2939e09a0809bbc5182a5bc/lib/guice-5.1.0-clas
        WARNING: Please consider reporting this to the maintainers of class com.google.inject.internal.aop.HiddenClassDefiner
        WARNING: sun.misc.Unsafe::staticFieldBase will be removed in a future release
```

may be there is an issue somewhere else (e.g. trying to use the maven wrapper instead of the maven installation as specified by the path). not sure. but we have taken the habit to remove the wrapper to avoid any confusion for the tooling.

--- a/devtools/maven/src/main/java/io/quarkus/maven/CreateProjectMojo.java
+++ b/devtools/maven/src/main/java/io/quarkus/maven/CreateProjectMojo.java
@@ -89,6 +89,12 @@ public class CreateProjectMojo extends AbstractMojo {
     @Parameter(property = "noCode", defaultValue = "false")
     private boolean noCode;
 
+    /**
+     * When true, do not include the Maven/Gradle wrapper
+     */
+    @Parameter(property = "noBuildToolWrapper", defaultValue = "false")
+    private boolean noBuildToolWrapper;
+
     @Parameter(property = "example")
     private String example;
 
@@ -313,6 +319,7 @@ public class CreateProjectMojo extends AbstractMojo {
                     .resourcePath(path)
                     .example(example)
                     .noCode(noCode)
+                    .noBuildToolWrapper(noBuildToolWrapper)
                     .appConfig(appConfig)
                     .data(data);
 

